.github/workflows/spell_check.yml

```
@@ -0,0 +1,36 @@
+name: Check Spelling
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  spelling:
+    name: Check Spelling
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'RT-Thread'
+    permissions:
+      contents: read
+      pull-requests: read
+      actions: read
+    steps:
+    - name: output ignore words info
+      run: |
+        echo "> If any words shown as incorrect are spelled correctly, please add them to .github/actions/spelling/allow.txt." >> $GITHUB_STEP_SUMMARY
+    - name: check spelling
+      id: spelling
+      uses: check-spelling/check-spelling@prerelease
```

看看v0.0.22怎么样 preelease分支看起来不是一个长久分支，最好用个发布版本。
```suggestion
      uses: check-spelling/check-spelling@v0.0.22
```

---

就是v0.0.22用不了，换成了v0.0.21看看😂